includes/class-assets.php

```diff
@@ -290,6 +290,8 @@ public function enqueue_admin_scripts( $force = false ) {
 
         wp_enqueue_style( 'wpbdp-js-select2-css' );
 
+		do_action( 'wpbdp_enqueue_admin_scripts' );
```

Comment:
Let’s add a doc block here that includes since. This will make it easier to know if we can use it in addons.